packages/client/__tests__/fields/__snapshots__/Text.spec.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This snapshot does not contain useful content. You need to pass in correct props and make sure it renders correctly.